### DISCUSSION TITLE
How to connect redux devtools to Recoil - A package to the rescue

### DISCUSSION THREAD

**Discussion by creotip** [facebookexperimental/Recoil#1508] [Show and tell]
Hi guys, here you can check a package I created:
https://recoil-gear.netlify.app/

It's a simple connector to redux devtools

**Comment by Cpt-Falcon** [comment]
Wow this is sweet thank you. The meta team should just hire you, I don't understand how this didn't already exist as part of support for recoil

**Reply by theohagos** [reply]
I also have something similar but you can also use replay and time travel:

https://www.npmjs.com/package/@theohagos/recoil-observer

**Comment by jbalatero** [comment]
The url: https://[recoil-gear.netlify.app](https://recoil-gear.netlify.app/)/ does not work

**Reply by theohagos** [reply]
why don't you try this https://www.npmjs.com/package/@theohagos/recoil-observer, it works really well
